tp1/main.cpp

```diff
@@ -40,7 +40,7 @@ void showData(const vector<int>& data) {
 int main (int argc, char** argv) {
 
   srand(time(0));
-  
+
```

Comment:
Ça pourrait être un bonne idée de pouvoir spécifier un seed par la ligne de commande, mais c'est out-of-scope.